### PR TITLE
Feature/profile ui refresh iteration 2

### PR DESCRIPTION
**Pull request by flexsurfer**
[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

### Summary:
[comment]: # (Summarise the problem and how the pull request solves it)
Profile UI refresh (iteration 2)

### Steps to test:
- Open Status
- Open your profile
- Edit profile
- Open someone profile

### Known issues:
Name field differs from the mockup because text-field component is very complex (label's floating animation) and changes are not worth it, so @andmironov  said it's permissible deviations
image
And also status field, because we have two fields, static text and edit area it's difficult to align them

### Not implemented in this PR:

- Information about status update 
- Chat with yourself 

[comment]: # (PRs will only be accepted if squashed into single commit.)
status: ready

### CODE REVIEW

File: src/status_im/profile/styles.cljs [hunk 1]
```diff
@@ -9,156 +10,128 @@
...
-(def profile-setting-item
-  (merge (ps-profile :profile-setting-item)
-         {:flex-direction :row
-          :align-items    :center}))
+(defstyle profile-setting-item
+  {:flex-direction :row
+   :align-items    :center
+   :padding-left  16
+   :padding-right 16
+   :ios            {:height        73}
```
Comment (rasom) [inline src/status_im/profile/styles.cljs]: again weird indentations

File: src/status_im/profile/styles.cljs [hunk 2]
```diff
@@ -183,11 +156,40 @@
...
+
+(defstyle profile-status-text
+  {:color        text1-color
+   :line-height  24
+   :ios          {:font-size      17
+                  :letter-spacing -0.2}
+   :android      {:font-size      16}})
+
+(defstyle edit-profile-status
+  {:background-color color-light-gray
```
Comment (rasom) [inline src/status_im/profile/styles.cljs]: indentations

**Comment by flexsurfer** [comment]
@rasom i mentioned this in the description

**Comment by rasom** [comment]
**[Android]**

screen shot 2017-03-28 at 16 52 42

1) wrong position
2) different height

**Comment by flexsurfer** [comment]
@rasom 
Android [1-3] i mentioned in the description, @andmironov will change the mockup if needed
4- we can implement in another issue , 
iOS  [1-2] i mentioned in the description, @andmironov will change the mockup if needed

**Comment by rasom** [comment]
Ok, thanks, @flexsurfer. I need to adjust my "blind robot reviewer mode". Please fix comments related to code and I will merge PR.

**Comment by flexsurfer** [comment]
@rasom done

**Pull request merged**